src/OrbitBase/FutureTest.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "OrbitBase/Future.h"
+#include "OrbitBase/Promise.h"
+
+namespace orbit_base {
+
+TEST(Future, IsValid) {
```

Comment:
In spirit of testing scenarios and not methods: rename this to Create?

Reply:
Sure. I'm very bad at naming tests. So more feedback like that is very much appreciated!